architecture/additional_concepts/authorization.adoc

```diff
@@ -273,14 +273,30 @@ ifdef::openshift-enterprise,openshift-origin[]
 === Updating Cluster Roles
 
 After any xref:../../install_config/upgrading/index.adoc#install-config-upgrading-index[{product-title} cluster
-upgrade], the recommended default roles may have been updated. See
+upgrade], the default roles are updated and automatically reconciled when the
+server is started. Additionally see
 xref:../../install_config/upgrading/manual_upgrades.adoc#updating-policy-definitions[Updating
-Policy Definitions] for instructions on getting to the new recommendations
+Policy Definitions] for instructions on getting other recommendations
 using:
 
 ----
 $ oc adm policy reconcile-cluster-roles
 ----
+
```

Comment:
Need to add a section ID before the section heading:

[[applying-custom-roles-and-permissions]]